activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter/src/test/java/org/activiti/starter/tests/TestConfiguration.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.activiti;
+package org.activiti.starter.tests;
```

Comment:
This class is no longer necessary. Please use the one from activiti-spring-boot-test.